pkg/apis/management.cattle.io/v3/tools_system_images.go

```diff
@@ -13,17 +13,17 @@ var (
 		AuthSystemImages     AuthSystemImages
 	}{
 		PipelineSystemImages: projectv3.PipelineSystemImages{
-			Jenkins:       m("rancher/pipeline-jenkins-server:v0.1.4"),
-			JenkinsJnlp:   m("jenkins/jnlp-slave:3.35-4"),
-			AlpineGit:     m("rancher/pipeline-tools:v0.1.15"),
-			PluginsDocker: m("plugins/docker:18.09"),
-			Minio:         m("minio/minio:RELEASE.2020-07-13T18-09-56Z"),
-			Registry:      m("registry:2"),
-			RegistryProxy: m("rancher/pipeline-tools:v0.1.15"),
-			KubeApply:     m("rancher/pipeline-tools:v0.1.15"),
+			Jenkins:       "rancher/pipeline-jenkins-server:v0.1.4",
+			JenkinsJnlp:   "rancher/mirrored-jenkins-jnlp-slave:3.35-4",
```

Comment:
These are not in latest RC images.txt. This won't add it to that will it ?

Reply:
The non without the "mirrored-" prefix is, `rancher/jenkins-jnlp-slave:3.35-4`, and yes it will replace that with the new one.